### PR TITLE
`isTokenAnImageConcept` feature added.

### PR DESCRIPTION
**Pull request by Rahgooy**
`isTokenAnImageConcept` feature added.

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala [hunk 1]
```diff
@@ -125,6 +125,12 @@ object MultiModalSpRLDataModel extends DataModel {
       getTokenDistance(first, second)
   }
 
+  val isTokenAnImageConcept = property(tokens) {
+    t: Token =>
+      ((tokens(t) <~ sentenceToToken <~ documentToSentence) ~> documentToImage ~> imageToSegment)
+        .map(_.getSegmentConcept) exists (x => t.getText.contains(x))
+  }
+
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:133]: Nice!

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:133]: We need to do some word embedding features here I guess as the next step.

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:133]: Yes, we need to use some kind of similarity(using wordnet or word-to-vector, ...)

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:133]: wordToVec I would say.

**Pull request merged**